scripts/cut_release_branch.py

```diff
@@ -35,6 +35,20 @@
 
 
 def new_version_type(arg, pattern=re.compile(r'\d\.\d\.\d')):
+    r"""Checks that the new version name matches the expected pattern.
```

Comment:
For the changes in this file and the next one, please put them in a different PR. You can have multiple PRs open at a time -- just make sure to do them on separate branches from develop. In general, one PR = one change.

(It's also fine to close this PR and create twoseparate new branches from develop if it's easier.)